runelite-client/src/main/java/net/runelite/client/plugins/masterfarmer/MasterFarmerOverlay.java

```diff
@@ -0,0 +1,78 @@
+package net.runelite.client.plugins.masterfarmer;
+
+import java.awt.Dimension;
+import java.awt.Graphics2D;
+import javax.inject.Inject;
+import net.runelite.api.Client;
+import net.runelite.api.NPC;
+import net.runelite.api.Point;
+import net.runelite.client.ui.overlay.Overlay;
+import net.runelite.client.ui.overlay.OverlayLayer;
+import net.runelite.client.ui.overlay.OverlayPosition;
+import net.runelite.client.ui.overlay.OverlayUtil;
+
+public class MasterFarmerOverlay extends Overlay
+{
+
+	private final Client client;
+	private final MasterFarmerPlugin plugin;
+	private final MasterFarmerConfig config;
+
+	@Inject
+	MasterFarmerOverlay(Client client, MasterFarmerPlugin plugin, MasterFarmerConfig config)
+	{
+		setPosition(OverlayPosition.DYNAMIC);
+		setLayer(OverlayLayer.ABOVE_SCENE);
+		this.plugin = plugin;
+		this.config = config;
+		this.client = client;
+	}
+
+	@Override
+	public Dimension render(Graphics2D graphics)
+	{
+		if (config.showOverlay())
+		{
+			for (NPC npc : client.getNpcs())
+			{
+				final String npcName = npc.getName();
+
+				if (npcName == null)
+				{
+					continue;
+				}
+
+				if (npc.getId() == 3257 || npc.getId() == 3258)
```

Comment:
reference the var MASTER_FARMER from NpcID class instead of 3257 as a magic number. same for 3258

Reply:
Fixed